packages/EasyStandard/composer.json

```diff
@@ -4,7 +4,8 @@
     "type": "library",
     "license": "MIT",
     "require": {
-        "php": "^7.2"
+        "php": "^7.2",
+        "symfony/dependency-injection": "5.1.10"
```

Comment:
@natepage here we have to use the fixed version because of the breaking changes introduced in the next versions:
```
70) EonX\EasyStandard\Tests\Sniffs\Methods\TestMethodNameSniff\TestMethodNameSniffTest::testProcessSucceedsIfNamespaceDoesNotHaveAllowedPatterns
Undefined index: serviceKeywords

/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyStandard/vendor/symplify/easy-coding-standard/src/Yaml/CheckerServiceParametersShifter.php:72
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyStandard/vendor/symplify/easy-coding-standard/src/Yaml/FileLoader/CheckerTolerantYamlFileLoader.php:21
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyStandard/vendor/symplify/easy-coding-standard/src/DependencyInjection/DelegatingLoaderFactory.php:48
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyStandard/vendor/symplify/easy-coding-standard/src/DependencyInjection/DelegatingLoaderFactory.php:25
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyStandard/vendor/symplify/easy-coding-standard/src/HttpKernel/EasyCodingStandardKernel.php:39
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyStandard/vendor/symfony/http-kernel/Kernel.php:644
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyStandard/vendor/symfony/http-kernel/Kernel.php:540
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyStandard/vendor/symfony/http-kernel/Kernel.php:780
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyStandard/vendor/symfony/http-kernel/Kernel.php:121
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyStandard/vendor/symplify/package-builder/src/Tests/AbstractKernelTestCase.php:118
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyStandard/vendor/symplify/package-builder/src/Tests/AbstractKernelTestCase.php:54
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyStandard/vendor/symplify/easy-coding-standard-tester/src/Testing/AbstractCheckerTestCase.php:53
```

Reply:
@roman-eonx ok no problem, thanks.